src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const everyDayCarRentCost = 40;
+  let discount = 0;
```

Comment:
Since 'discount' is not reassigned after its initial definition, it's better to use 'const' instead of 'let'. However, if you apply the suggestion to use 'return' statements inside the 'if' blocks, you won't need the 'discount' variable at all.